sdk/documenttranslator/ai-document-translator-rest/test/text.spec.ts

```diff
@@ -0,0 +1 @@
+export const x = "foo";
```

Comment:
I guess this file was accidentally checked in?

Reply:
yes thanks for catching it :)